spring-boot-autoconfigure/src/main/java/org/axonframework/boot/autoconfig/RoutingStrategyAutoConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2010-2017. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package org.axonframework.boot.autoconfig;
+
+import org.axonframework.commandhandling.distributed.AnnotationRoutingStrategy;
+import org.axonframework.commandhandling.distributed.RoutingStrategy;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Auto configuration for {@link RoutingStrategy}.
+ *
+ * @author Milan Savic
+ * @since 3.1.1
+ */
+@Configuration
+@AutoConfigureAfter(JpaAutoConfiguration.class)
```

Comment:
This configuration doesn't have a dependency on Jpa, so I guess this requirement could be removed.
I see it was part of the other auto configuration classes (and now removed). That's fine, those didn't depend on JPA either.